components/eamxx/src/share/io/scream_io_control.hpp

```diff
@@ -39,6 +39,30 @@ struct IOControl {
                                         ts.get_time()==next_write_ts.get_time());
   }
 
+  void set_frequency_units (const std::string& freq_unit) {
+    if (freq_unit=="none" or freq_unit=="never") {
+      frequency_units = freq_unit;
+    } else if (freq_unit=="nstep" or freq_unit=="nsteps") {
```

Comment:
Do we need to support the version without the `s` at the end? I find the one with the s to be grammatically more correct. Unless e3sm or eam already allow the "singular" version, I would not support it.

Reply:
yes, E3SM/EAM already support the one w/o the "s"